stac_model/input.py

```diff
@@ -71,9 +71,10 @@ class ValueScalingProcessingExpression(ProcessingExpression):
         ValueScalingScale,
         ValueScalingProcessingExpression,
     ]
-]
+    | None
+  ]  # noqa: E501
```

Comment:
Doesn't look like a false positive, should actually be indented by 4 spaces, not 2.

The `| None` should be not be there at all since it's already wrapped in a `Optional`.

Reply:
removed the | None, `make format` prefers using python built in operators instead of typing classes like Optional.

The indentation passes make lint, doesn't seem like an issue to me if it passes ruff linting.

Reply:
The indentation is incorrect. The error is ignored because of `noqa: E501`. It must be removed.

In should be like this: 

```python
ValueScalingObject: TypeAlias = Optional[
    Union[
        ValueScalingMinMax,
        ValueScalingZScore,
        ValueScalingClip,
        ValueScalingClipMin,
        ValueScalingClipMax,
        ValueScalingOffset,
        ValueScalingScale,
        ValueScalingProcessingExpression,
    ]
]  # <== no spaces before ']'
```

Reply:
after removing noqa, `make format` prefers no spaces, I suggest we go with this, looks clean and would rather not fight make format and standardize on ruff's codestyle.